samples/net/zperf/README.rst

```diff
@@ -12,7 +12,14 @@ evaluate network bandwidth.
 Features
 *********
 
-- Compatible with iPerf_2.0.5.
+- Compatible with iPerf_2.0.9. Note that UDP uploader (client) testing did
+  not work properly in iPerf_2.0.13 and error message like this is printed
```

Comment:
```suggestion
  not work properly in iPerf_2.0.13: an error message like this is printed
```

Reply:
BTW, can you mention if this is a problem with iPerf or with our code (and will be fixed later)?

Reply:
Impossible to know, the iPerf protocol is not really documented, we are just reverse engineering this from wireshark output. Probably we would need to change zephyr implementation but I will leave this for future.